README.md

```diff
@@ -1,44 +1,52 @@
 # UMA Protocol
 
-## Contact
+<div style="text-align:center"><img src="./documentation/Logo.png" /></div>
 
-- [Slack](https://umaprotocol.slack.com): to join, use this
-  [invite link](https://join.slack.com/t/umaprotocol/shared_invite/enQtNTk4MjQ4ODY0MDA1LTM4ODg0NGZhYWZkNjkzMDE4MjU0ZGFlYWQzZTFiZWFlZjI2NDE4OGI2NWY3OTdhYjYyZjg0MjAzMTgwODVhZTE).
-  Please use Slack for all technical questions and discussions.
-- [Email](mailto:hello@umaproject.org): for anything non-technical.
+[![<UMAprotocol>](https://circleci.com/gh/UMAprotocol/protocol.svg?style=shield)](https://app.circleci.com/pipelines/github/UMAprotocol/protocol)
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/umaprotocol/voting)
 
-## Documentation
+![GitHub](https://img.shields.io/github/license/UMAprotocol/protocol)
+![GitHub last commit](https://img.shields.io/github/last-commit/UMAprotocol/protocol)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/UMAprotocol/protocol)
+![GitHub contributors](https://img.shields.io/github/contributors-anon/UMAprotocol/protocol)
+
+[![Generic badge](https://img.shields.io/badge/homepage-view-red.svg)](https://umaproject.org/)
+[![Generic badge](https://img.shields.io/badge/slack-join-green.svg)](https://join.slack.com/t/umaprotocol/shared_invite/zt-7mtxxds5-OIhE~q_WkwGCVNrq0~G~rg)
+[![Generic badge](https://img.shields.io/badge/send-email-blue.svg)](mailto:hello@umaproject.org)
+![Twitter Follow](https://img.shields.io/twitter/follow/UMAprotocol?label=follow%20%40UMAprotocol&style=social)
```

Comment:
This seems to link to a page with just this image on it -- maybe we should link this image to twitter?

Reply:
done